activerecord/test/cases/associations_test.rb

```diff
@@ -241,6 +241,19 @@ def test_has_many_association_from_a_model_with_query_constraints_different_from
     assert_equal(expected_comments.sort, comments.sort)
   end
 
+  def test_foreign_key_is_in_ignored_columns
+    comment_class = Class.new(ActiveRecord::Base) do
+        self.table_name = "comments"
+        self.ignored_columns = %w[post_id]
+
+        belongs_to :post
+      end
```

Comment:
Is there a reason not to use just two spaces?
```suggestion
    comment_class = Class.new(ActiveRecord::Base) do
      self.table_name = "comments"
      self.ignored_columns = %w[post_id]

      belongs_to :post
    end
```